plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/status/StatusPlexusResource.java

```diff
@@ -74,7 +74,7 @@ public String getResourceUri() {
 
   @Override
   public PathProtectionDescriptor getResourceProtection() {
-    return new PathProtectionDescriptor(getResourceUri(), "authcNxBasic,perms[nexus:status]");
+    return new PathProtectionDescriptor(getResourceUri(), "authcBasic,perms[nexus:status]");
```

Comment:
IIRC authcNxBasic was created to prevent basic-auth dialogs to show up when attempting to login....